openfe/setup/atommapping.py

```diff
@@ -13,17 +15,27 @@ class AtomMapping:
 
     Attributes
     ----------
-    mol1, mol2 : rdkit.Mol
+    mol1, mol2 : Molecule
       the two Molecules in the mapping
     mol1_to_mol2 : dict
       maps the index of an atom in either molecule **A** or **B** to the other.
       If this atom has no corresponding atom, None is returned.
 
     """
-    mol1: RDKitMol
-    mol2: RDKitMol
+    mol1: Molecule
+    mol2: Molecule
     mol1_to_mol2: Dict[int, int]
 
+    def __init__(self, mol1: Union[Molecule, RDKitMol],
```

Comment:
So currently AtomMappers work off an RDKit Mol, which seems correct, and they return an `AtomMapping`.  I think it's probably easiest to do the conversion from rdkit to hashable Molecule here, rather than have the boilerplate in every single `AtomMapper`.